konig-schemagen/src/main/java/io/konig/schemagen/sql/RdbmsShapeGenerator.java

```diff
@@ -104,12 +99,11 @@ public Shape createRdbmsShape(Shape shape) throws RDFParseException, IOException
 	 * @return true if the supplied shape contains fields that need to be renamed or any nested shapes.
 	 */
 	private boolean accept(Shape shape) {
-		
-		if (!SqlTableGeneratorUtil.isValidRdbmsShape(shape)) {
+		if (!isValidRdbmsShape(shape)  || !shape.getProperty().isEmpty() ) {
```

Comment:
I don't think we should require that the shape have no properties.  We might want to allow the user to override the property constraints that get generated by default.  Let's leave this condition as is for now, but we may want to revisit in the future.

Reply:
The idea behind adding this condition is , when the properties have been added to the rdbms shape for a child shape, through createOneToManyChildShape  method, it should not be added again to the same rdbms shape through createRdbmsShape considering it as a parent shape.  We need to handle this scenario in some other way in case rdbms shape can have properties in the excel itself.